src/components/ble/NotificationManager.h

```diff
@@ -57,6 +62,7 @@ namespace Pinetime {
       size_t NbNotifications() const;
 
     private:
+      Controllers::DateTime& dateTimeController;
```

Comment:
we can even store a `const` reference, to make it explicit, that a notification can't change the date-time, just get the current date and time

```suggestion
      const Controllers::DateTime& dateTimeController;
```